cdap-docs/reference-manual/source/http-restful-api/introduction.rst

```diff
@@ -126,10 +126,14 @@ Status Codes
      - The request had a combination of parameters that is not recognized
    * - ``401``
      - ``Unauthorized``
-     - The request did not contain an authentication token
+     - The request did not contain an authentication token; requests can fail due to a
```

Comment:
Can we say `requests can fail due to a lack of authentication,` and then include the link, as it links to a section describing using authentication.